### PR TITLE
Add support for translation using OpenAI API

### PR DESCRIPTION
**Pull request by vusallyv**
### All Submissions:

- [X] Are tests passing? (From the root-level of the repository please run `pip install tox && tox`)
- [X] I have added or updated a test to cover the changes proposed in this Pull Request
- [X] I have updated the documentation to cover the changes proposed in this Pull Request

### CODE REVIEW

File: rosetta/translate_utils.py [hunk 1]
```diff
@@ -167,3 +170,31 @@ def translate_by_google(
...
+    """
+    Translate text using OpenAI's GPT-3.5-turbo-instruct engine.
+    param text: The text to translate.
+    param from_language: The language of the text.
+    param to_language: The language to translate the text into.
+    param api_key: The OpenAI API key.
+    return: The translated text.
+    """
+
+    import openai
```
Comment (mbi) [inline rosetta/translate_utils.py]: What library is this? :pray:

Comment (vusallyv) [reply rosetta/translate_utils.py]: https://pypi.org/project/openai/0.28.0/

This library provides convenient access to the OpenAI API

Comment (mbi) [reply rosetta/translate_utils.py]: Any good reason to mention 0.28.0, when the latest version is 1.35.3?

Comment (vusallyv) [reply rosetta/translate_utils.py]: Upgraded 🙏

Comment (mbi) [reply rosetta/translate_utils.py]: Awesome, thanks.

**Pull request merged**